presto-docs/src/main/sphinx/troubleshoot/deploy.rst

```diff
@@ -0,0 +1,72 @@
+================
+Deploying Presto
+================
+
+.. contents::
```

Comment:
In the screen shot, it starts with just a list of links, maybe include a header or something like "Common Deployment Errors" above